packages/server/src/config.ts

```diff
@@ -98,6 +98,7 @@ export interface MedplumDatabaseConfig {
   password?: string;
   ssl?: MedplumDatabaseSslConfig;
   queryTimeout?: number;
+  maxInstanceConnections?: number;
```

Comment:
We should probably give this a name that clearly indicates "database connections".  We've been trending toward longer and more explicit config setting names.

```suggestion
  maxDatabaseConnectionsPerServer?: number;
```